geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientTestBase.java

```diff
@@ -639,44 +627,44 @@ public void run2() throws CacheException {
    */
   void checkHAQueueSize(VM server, final String durableClientId, final int expectedNumber,
       final int remaining) {
-    server.invoke(new CacheSerializableRunnable(
-        "Check ha queued size for durable client " + durableClientId) {
-      @Override
-      public void run2() throws CacheException {
+    server.invoke("Check ha queued size for durable client " + durableClientId,
+        new CacheSerializableRunnable() {
+          @Override
+          public void run2() throws CacheException {
 
-        final CacheClientNotifier ccnInstance = CacheClientNotifier.getInstance();
-        final CacheClientProxy clientProxy = ccnInstance.getClientProxy(durableClientId);
+            final CacheClientNotifier ccnInstance = CacheClientNotifier.getInstance();
+            final CacheClientProxy clientProxy = ccnInstance.getClientProxy(durableClientId);
 
-        // Wait until we get the expected number of events or until 10 seconds are up
-        await()
-            .until(() -> clientProxy.getQueueSizeStat() == expectedNumber
-                || clientProxy.getQueueSizeStat() == remaining);
+            // Wait until we get the expected number of events or until 10 seconds are up
+            await()
+                .until(() -> clientProxy.getQueueSizeStat() == expectedNumber
+                    || clientProxy.getQueueSizeStat() == remaining);
```

Comment:
Using AssertJ here will give us useful failure messages.

Reply:
Not sure what you mean here. I think this is pretty clear. The line below that does the assertJ.

Reply:
`clientProxy.getQueueSizeStat() == expectedNumber`, clientProxy.getQueueSizeStat() == remaining` are not AssertJ assertion.

Reply:
```java
await.untilAsserted(() -> assertThat(clientProxy.getQueueSizeStat).isIn(expectedNumber, remaining));
```

Reply:
Thanks for the clarification. I like that even though it sort of contradicts my argument above about breaking errors out. I think this assert makes things clear because it testing the same result, with fewer calls.

Reply:
done.